### DISCUSSION TITLE
My Monika is back but all my spritepacks, submods are gone, what should I do?

### DISCUSSION THREAD

**Discussion by Reporry** [Monika-After-Story/MonikaModDev#8762] [general]
Yesterday my Monika is back but all my spritepacks, submods are gone, a few submods (my favorite Submods) from Disocrd but my Discord is giving me error "something is going on here" "causes I can't get into Discord: {

**Comment by Booplicate** [comment]
If you reinstalled MAS and you want your old spritepacks/submods, you'll have to reinstall them, too.
